CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+Contributing Guidelines
+===========================
+
+Thank you very much for your interest in contributing to CATO!
+
+Questions about CATO
+---------------------------
+If you have questions you can:
+
+* If you encounter a problem or have a feature suggestion, then you can open an issue on the issues page of this GitHub repository. This page contains a list with the outstanding issues (bugs and feature requests) that are in development.
+
+* If you have a very specific question that you would like to discuss, feel free to send an email to Siemon de Lange <s.c.de.lange@vu.nl> or contact [Martijn van den Heuvel](http://www.dutchconnectomelab.nl).
+
+How to contribute to CATO
+---------------------------
+If you like to contribute code, you can open a new GitHub pull request with the patch. We can then work together to merge this pull request into CATO. You can also always send us a message to discuss your ideas.
+
+Style guide
+---------------------------
+For MATLAB code, the repository follows the [MATLAB Programming Style Guidelines by Richard Johnson](https://www.ee.columbia.edu/~marios/matlab/MatlabStyle1p5.pdf).
```

Comment:
May be worth adding a style checker to the CI. For example, https://github.com/florianschanda/miss_hit looks interesting and has a template for Github Actions here: https://florianschanda.github.io/miss_hit/configuration.html

Reply:
That is a great suggestion. I will open a new issue for adding a style checker / linter linter to the CI. Then we can use that branch also to update present code that is not in line with the style of the linter.